documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Please don't <a href="https://www.ibm.com/au-en/cloud/learn/exploratory-data-analysis" target="_blank">copy text</a> from sites without permission. If I've missed <em>anything</em> that has come from another site, remove it immediately, or flag it so I can review the license.</p><p> </p><p>This section is out of place anyway. We have just prepared a dataset for training and are about to train a model. Explaining EDA is confusing and would make more sense in a section where we explore the data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>